Note support for Concealed Authentication (#4)
bemasc/h3-webtrans: draft-various-httpbis-h3-webtrans.md

# Security Considerations

> Authentication between endpoints is crucial for secure deployment of H3-WT.  Depending on the use case, authentication of one or both participants may be needed.  This specification is compatible with many suitable techniques, including TLS server authentication, Mutual TLS, HTTP Client Authentication (when using WebTransport over HTTP), and Capability URLs.

Reviewer: ```suggestion
Authentication between endpoints is crucial for secure deployment of H3-WT.  Depending on the use case, authentication of one or both participants may be needed.  This specification is compatible with many suitable techniques, including TLS server authentication, TLS client authentication (commonly referred to as mutual TLS), HTTP Authentication, and Capability URLs.
```
I don't think HTTP client authentication necessary mandates WebTransport over HTTP. Nothing stops implementors require relevant HTTP authentication within H3-WT.

Author: I think there's an important but subtle distinction here between "transport authentication" of the H3-WT session and "request authentication" of the requests within it (which may have the roles reversed!).  I've rewritten this a bit for clarity.

> If the WebTransport protocol exposes a TLS Exported Authenticator capability, participants MAY use it to enable Secondary Certificate Authentication {{?I-D.ietf-httpbis-secondary-server-certs}} or HTTP Concealed Authentication {{?I-D.ietf-httpbis-unprompted-auth}} in the H3-WT session.

Reviewer: Looks great!
